src/app/components/module-index-page/module-index-page.component.ts

```diff
@@ -53,33 +66,61 @@ export class ModuleIndexPageComponent implements OnInit {
                if (response != null) {
                   this.parseContentResponse(response, module);
                } else {
-                  alert('Response was null');
+                  this.toastr.error('Response was null');
                }
             },
             (response) => {
-               alert("Failed to request contents");
+               this.toastr.error('Failed to request contents');
+               
             },
+            /* display module's contents when done loading */
             () => { this.contentVisible.set(module, true); }
          )
-      } else {
+      }
 
+      else {
+         /* Toggle visibility of Module's list of Content
+         * using ModuleStoreServices's map of Module->Boolean */
          this.contentVisible.set(module, !this.contentVisible.get(module));
       }
    }
 
    /**
-     * Sort the content list order by title,
-     * insert into Module->List<Content> Map
-     * @param response
-     * @param module
-     */
+    * Sort the content list order by title
+    * Insert into Module->List<Content> Map
+    * @param response Available content
+    * @param module Tags/modules for content
+    */
    parseContentResponse(response: Content[], module: Module) {
 
+      /* sort contents by their title */
       let sortedResponse = response.sort(
          (a, b) => { return a.title < b.title ? -1 : 1 }
       )
 
       this.moduleContents.set(module, sortedResponse);
    }
 
+   /**
+    * Description - removes the content from the specified module. It will also send a request to decouple the link between content and module.
+    * @param content - the content being removed
+    * @param module - the module the content is being removed from
+    */
+   removeContentFromModuleIndex() {
+      //looks through the array of links that belongs to content and splices out the module/tag if it finds one.
```

Comment:
remove non documentation comments here

Reply:
and in module-creator-page.component.css, module-index-page.component.css, etc.

Reply:
Allen's already fixing it. Please note that many of the things that appear to be changing in components not supposed to be covered by our pull requests that appear to be are in fact a result of odd back-merging from dev into our branch and having to handle odd changes that aren't in fact in dev that we're not sure exactly when or how they occurred.

Reply:
It is done.